applications/callflow/src/module/cf_privacy.erl

```diff
@@ -35,10 +35,19 @@ update_call(CaptureGroup, {'ok', Call}) ->
                 }
                 ,{fun whapps_call:kvs_store/3, 'cf_privacy', 'true'}
                 ,{fun whapps_call:set_caller_id_name/2
-                  ,whapps_config:get_non_empty(<<"callflow">>, <<"privacy_name">>, <<"anonymous">>)
+                  ,default_cid_number
```

Comment:
This atom breaks things. This becomes a 3-tuple, but your function is arity 2. I don't think you meant to leave this atom here?